README.md

```diff
@@ -37,3 +37,21 @@ operator and seccomp support in Kubernetes in general:
 ![mind-map](.github/mindmap.svg)
 
 [7]: https://app.mindmup.com
+
+## Community, discussion, contribution, and support
+
+Learn how to engage with the Kubernetes community on the [community
+page](http://kubernetes.io/community/).
+
+You can reach the maintainers of this project at:
+
+- [Slack](http://slack.k8s.io/)
```

Comment:
Shall we add the seccomp-operator channel link here?

Reply:
Yes, I’ll add it. :)